cypress/integration/userCanSearchForASong.feature.js

```diff
@@ -14,7 +14,21 @@ describe('User can search a song to post by keyword', () => {
       url: 'http://localhost:3000/api/tracks?q=**',
       response: { tracks: tracks },
     })
+    cy.route({
+      method: 'POST',
+      url: 'http://localhost:3000/auth/sign_in',
+      response: 'fx:user_login_with_devise_credentials.json',
+    })
+    cy.route({
+      method: 'GET',
+      url: 'http://localhost:3000/auth/validate_token**',
+      response: 'fx:user_login_with_devise_credentials.json',
+    })
```

Comment:
Do we need this route? Can you try running the test without it?

Reply:
we don't need it yet, good catch